plugin/editorconfig.py

```diff
@@ -0,0 +1,42 @@
+try:
+    try:
+        import vim
+        import sys
+    except:
+        vim.command('let l:ret = 2')
+        raise
+
+    try:
+        sys.path.insert(0, vim.eval('a:editorconfig_core_py_dir'))
+
+        import editorconfig
+        import editorconfig.exceptions as editorconfig_except
+
+    except:
+        vim.command('let l:ret = 3')
+        raise
```

Comment:
I noticed you added a "raise" here. However, if an exception occurs, "del sys.path[0]" would not execute. Can you fix it please?